api/casauth.go

```diff
@@ -0,0 +1,75 @@
+package api
+
+import (
+	"github.com/wtg/shuttletracker/auth"
+	"github.com/wtg/shuttletracker/database"
+	"github.com/wtg/shuttletracker/log"
+	"net/url"
+
+	gc "gopkg.in/cas.v2"
+
+	"net/http"
+	"strings"
+)
+
+// CasClient stores the local cas client and an instance of the database
+type CasClient struct {
+	cas auth.AuthenticationService
+	db  database.Database
+}
+
+func CreateCasClient(url *url.URL, db database.Database) (*CasClient){
```

Comment:
I think this should be called `CreateCASClient` (https://github.com/golang/go/wiki/CodeReviewComments#initialisms)